tacker/common/utils.py

```diff
@@ -299,3 +300,11 @@ def cpu_count():
         return multiprocessing.cpu_count()
     except NotImplementedError:
         return 1
+
+
+def is_auth_uri_v3(auth_url):
+    return (re.match('.+v3/?$', auth_url) is not None)
```

Comment:
We can do the same like below:

```python
def get_keystone_client(auth_url):
    if re.match('.+v3/?$', auth_url) is not None:
        from keystoneclient.v3 import client as ks_client
    else:
        from keystoneclient.v2_0 import client as ks_client
    return ks_client
```

And inside the other files we can get the Keystone client with:

```python
ks_client = common_utils.load_keystone_client(cfg.CONF.keystone_authtoken.auth_uri)
```

Similar to the Implementation in the "upstream" Tacker https://github.com/openstack/tacker/blob/master/tacker/vnfm/keystone.py#L32 this would provide a single source (makes debugging easier).